### PR TITLE
Simplify logic in SCI plugin

### PR DESCRIPTION
**Pull request by jmcook1186**
### Types of changes

- [x] Enhancement (project structure, spelling, grammar, formatting)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### A description of the changes proposed in the Pull Request
Applies simplified logic to `SCI` plugin and updates associated documentation, tests and manifests.

### CODE REVIEW

File: src/builtins/sci/index.ts [hunk 1]
```diff
@@ -7,176 +7,84 @@ import {validate, allDefined} from '../../util/validations';
...
-    const factor = getFunctionalUnitConversionFactor(input);
-
-    if (!input['functional-unit-time']) {
+  const execute = (inputs: PluginParams[]): PluginParams[] => {
+    return inputs.map(input => {
+      const safeInput = validateInput(input);
+      let sci = 0;
+      if (safeInput['carbon'] > 0) {
+        sci = safeInput['carbon'] / input[globalConfig['functional-unit']];
+      }
```
Comment (narekhovhannisyan) [inline src/builtins/sci/index.ts:45]:
```ts
const sci = safeInput['carbon'] > 0 ? safeInput['carbon'] / input[globalConfig['functional-unit']] : 0
```

File: src/builtins/sci/index.ts [hunk 2]
```diff
@@ -7,176 +7,84 @@ import {validate, allDefined} from '../../util/validations';
...
-      input[functionalUnit] !== 'none' &&
-      input[functionalUnit] !== ''
-      ? input[functionalUnit]
-      : 1;
-  };
+  // const convertSciToTimeUnit = (
+  //   sciPerSecond: number,
+  //   functionalUnitTime: { unit: string; value: number }
+  // ): number => {
+  //   const conversionFactor = TIME_UNITS_IN_SECONDS[functionalUnitTime.unit];
```
Comment (narekhovhannisyan) [inline src/builtins/sci/index.ts:60]: please remove unused comment

**Review by narekhovhannisyan — CHANGES_REQUESTED** [review]
please fix the requested changes above

File: manifests/examples/nesting.yml [hunk 3]
```diff
@@ -37,7 +37,14 @@ initialize:
       method: Sci
       global-config:
         functional-unit: "requests"
-        functional-unit-time: "1 minute"
+    "sum-carbon":
+      path: "@grnsft/if-plugins"
```
Comment (manushak) [inline manifests/examples/nesting.yml:41]: shouldn't the path be 'builtin'?

File: src/builtins/sci/README.md [hunk 4]
```diff
@@ -108,23 +39,18 @@ const sci = Sci({'functional-unit': 'requests'});
 const results = await sci.execute(
   [
     {
-      'carbon-operational': 0.02,
-      'carbon-embodied': 5,
+      'carbon': 5',
```
Comment (manushak) [inline src/builtins/sci/README.md:42]: please, remove `'` after 5

File: src/builtins/sci/index.ts [hunk 5]
```diff
@@ -7,176 +7,75 @@ import {validate, allDefined} from '../../util/validations';
...
-   */
-  const convertSciToTimeUnit = (
-    sciPerSecond: number,
-    functionalUnitTime: {unit: string; value: number}
-  ): number => {
-    const conversionFactor = TIME_UNITS_IN_SECONDS[functionalUnitTime.unit];
+    const validatedConfig = validateConfig(globalConfig);
 
-    if (!conversionFactor) {
+    if (!(validatedConfig['functional-unit'] in input)) {
```
Comment (manushak) [inline src/builtins/sci/index.ts:61]: please, also check if the value of `input[validatedConfig['functional-unit']]` is > 0

File: manifests/examples/nesting.yml (hunk 3 above)
Comment (jmcook1186) [reply manifests/examples/nesting.yml:41]:
```suggestion
      path: "builtin"
```

File: src/builtins/sci/README.md (hunk 4 above)
Comment (jmcook1186) [reply src/builtins/sci/README.md:42]:
```suggestion
      'carbon': 5'
```

File: src/builtins/sci/index.ts (hunk 5 above)
Comment (jmcook1186) [reply src/builtins/sci/index.ts]: done

File: manifests/examples/pipeline-teads-sci.yml [hunk 6]
```diff
@@ -26,8 +26,15 @@ initialize:
       path: "builtin"
       method: Sci
       global-config:
-        functional-unit: ""
-        functional-unit-time: "1-day"
+        functional-unit: "component"
+    "sum-carbon":
+      path: "@grnsft/if-plugins"
```
Comment (manushak) [inline manifests/examples/pipeline-teads-sci.yml:31]: please, rename here as well

File: manifests/examples/pipeline-with-aggregate.yml [hunk 7]
```diff
@@ -26,7 +26,14 @@ initialize:
       method: Sci
       global-config:
         functional-unit: "requests"
-        functional-unit-time: "1 minute"
+    "sum-carbon":
+      path: "@grnsft/if-plugins"
```
Comment (manushak) [inline manifests/examples/pipeline-with-aggregate.yml:30]: and here

File: manifests/examples/pipeline-with-mocks.yml [hunk 8]
```diff
@@ -39,12 +39,19 @@ initialize:
     "sci-o":
       path: "@grnsft/if-plugins"
       method: SciO
+    "sum-carbon":
+      path: "@grnsft/if-plugins"
```
Comment (manushak) [inline manifests/examples/pipeline-with-mocks.yml:43]: and here

File: manifests/examples/pipeline-teads-sci.yml (hunk 6 above)
Comment (jmcook1186) [reply manifests/examples/pipeline-teads-sci.yml:31]:
```suggestion
      path: "builtin"
```

File: manifests/examples/pipeline-with-aggregate.yml (hunk 7 above)
Comment (jmcook1186) [reply manifests/examples/pipeline-with-aggregate.yml:30]:
```suggestion
      path: "builtin"
```

File: manifests/examples/pipeline-with-mocks.yml (hunk 8 above)
Comment (jmcook1186) [reply manifests/examples/pipeline-with-mocks.yml:43]:
```suggestion
      path: "builtin"
```

**Review by narekhovhannisyan — APPROVED** [review]

**Pull request merged**